client/credentials_test.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"crypto/sha1"
+	"hash"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCredentials(t *testing.T) {
+
+	assert := assert.New(t)
+
+	credentials := &Credentials{
+		ClientID:    "tester",
+		AccessToken: "no-secret",
+	}
+
+	var credentialsTests = []struct {
```

Comment:
We have only one test case and index is hard coded, there is no point in have a array here.